### ISSUE TITLE
End up in with Task list after task completed from contact profile

### ISSUE THREAD

**Issue by abbyad** [medic/medic-webapp#2226]
When selecting a task from the Contact profile we end up in the Task tab. Given what happens in desktop mode I understand why this occurs, but in mobile mode it seems odd to end up Tasks list. I would have expected to remain in the Contacts profile. Along with #1975 and #1565, we should look at where we end up after completing forms based on where we came from.

**Comment by abbyad** [comment]
This is a symptom of https://github.com/medic/medic-webapp/issues/1368

**Comment by nice-snek** [comment]
hi frand @alxndrsn, please [triage](https://github.com/medic/medic-docs/blob/master/md/dev/workflow.md#triaging-old-issues) before the end of this sprint.
```
--..,_                        _,.--.
       ''.'.                .''__ o  ';__.
          '.'.            .'.''  '---''  '
            '.''--....--''.'
              ''--....--''
```

**Comment by alxndrsn** [comment]
Seems reasonable to keep this, but not high priority.  @sglangevin please close if this would be bad behaviour

**Comment by sglangevin** [comment]
Yes, let's keep this. We're hoping to do some app navigation updates and this is near the top of that list but not prioritized just yet.

**Comment by browndav** [comment]
This and the collection of associated issues appear as if they're blocked on design or have been superseded by work toward the v3.1 update. @sglangevin or @diannakane: if you have a chance, could you assist in reviewing?

**Comment by sglangevin** [comment]
This should be resolved with the v3.1 update. Closing.
